### PR TITLE
Update RnLdk.swift

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: ios/RnLdk.swift [hunk 1]
```diff
@@ -106,66 +106,76 @@ class MyFilter: Filter {
...
             } catch {
-                reject("start", "Failed",  error);
-                return;
+                reject("start", "Failed",  error)
+                return
             }
         } else {
-            channel_manager_constructor = ChannelManagerConstructor(network: LDKNetwork_Bitcoin, config: userConfig, current_blockchain_tip_hash: hexStringToByteArray(blockchainTipHashHex), current_blockchain_tip_height: UInt32(truncating: blockchainTipHeight), keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor!, router: nil, tx_broadcaster: broadcaster, logger: logger);
+            channel_manager_constructor = ChannelManagerConstructor(network: LDKNetwork_Bitcoin, config: userConfig, current_blockchain_tip_hash: hexStringToByteArray(blockchainTipHashHex), current_blockchain_tip_height: UInt32(truncating: blockchainTipHeight), keys_interface: keysInterface, fee_estimator: feeEstimator, chain_monitor: chain_monitor, router: nil, tx_broadcaster: broadcaster, logger: logger)
```
Comment (Overtorment) [inline ios/RnLdk.swift:162]: how dod u manage to get rid of `!` here?  i think xcode bitches that it might be null or smth

**Review by Overtorment — APPROVED** [review]
i trust everything works?

File: ios/RnLdk.swift (hunk 1 above)
Comment (marcosrdz) [reply ios/RnLdk.swift:162]: I used guard

**Comment by marcosrdz** [comment]
Didnt encounter issues on example app

**Pull request merged**